src/credentials/FabricTable.cpp

```diff
@@ -1715,7 +1729,7 @@ CHIP_ERROR FabricTable::AddNewPendingFabricCommon(const ByteSpan & noc, const By
 
     // Notify that NOC was added (at least transiently)
     *outNewFabricIndex = fabricIndexToUse;
-    NotifyFabricUpdated(fabricIndexToUse);
+    NotifyFabricAdded(fabricIndexToUse);
```

Comment:
Please add unit test for tbis

Reply:
Added it to `SecurePairingHandshakeTestCommon`

Reply:
Added